examples/player/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="../build.js"></script>
+    <script src="../common/shaders/skyGradient.js"></script>
+
+<script>
+// any extras would go here
+
+
+AFRAME.registerComponent('auto-hand-controls', {
+//  dependencies: ['tracked-controls'], // to make sure the system is present
+
+  init: function () {
+    // to make sure the tracked-controls system is present, put it on the scene (NOT an entity)
+    this.el.sceneEl.setAttribute('tracked-controls', '');
+
+    this.tick = AFRAME.utils.throttleTick ?
+      AFRAME.utils.throttleTick(this.throttledTick, 1000, this) :
+      this.throttledTick.bind(this);
+  },
+
+  throttledTick: function (t, dt) {
+    var hands = [];
+    var singlehand = 'right';
+    var isPresent = true;
+    if (AFRAME.utils.trackedControls.isControllerPresent(this.el.sceneEl, undefined, {})) {
+      hands.push('left');
+      hands.push('right');
+    } else
+    if (AFRAME.utils.gearvrControls && AFRAME.utils.gearvrControls.isControllerPresent()) {
+      hands.push(singlehand);
+    } else {
+      isPresent = false;
+    }
+    if (this.isPresent === isPresent) { return; }
+    this.isPresent = isPresent;
+    if (isPresent) {
+      this.hand = this.hand || {};
+      hands.forEach(function (hand) {
+        var handEl = this.hand[hand] = document.createElement('a-entity');
+        handEl.setAttribute('id', hand + '-hand');
+        handEl.setAttribute('hand-controls', hand);
+        if (hand === singlehand && AFRAME.utils.gearvrControls) { handEl.setAttribute('gearvr-controls', ''); }
+        this.el.appendChild(handEl);
+      }, this);
+      this.el.emit('lookmanewhands', {hands: this.hand});
+    } else
+    if (this.hand) {
+      Object.keys(this.hand).forEach(function (hand) {
+        var handEl = this.hand && this.hand[hand];
+        if (handEl) {
+          delete this.hand[hand];
+          if (handEl.parentNode) { handEl.parentNode.removeChild(handEl); }
+        }
+      }, this);
+      this.el.emit('lookmanohands', {});
+    }
+  }
+});
+
+AFRAME.registerComponent('auto-hand-teleport-controls', {
+    dependencies: ['auto-hand-controls'],
+
+    init: function () {
+        this.el.addEventListener('lookmanewhands', this.onNewHands.bind(this));
+    },
+
+    onNewHands: function (evt) {
+        var hands = evt.detail && evt.detail.hands;
+        if (hands) {
+            Object.keys(hands).forEach(function (key) {
+                if (hands.hasOwnProperty(key)) {
+                    hands[key].setAttribute('teleport-controls', 'collisionEntity', '#ground');
+                    hands[key].setAttribute('teleport-controls', 'playerEntity', '#player');
+                }
+            });
+        }
+    }
+});
+
+// FIXME: broken!
```

Comment:
that Object.map incantation is boilerplate to check that the object/map in question (hands map passed in evt.detail) actually has the key and it is not some intrinsic or inherited property.  so when passed a map with hand as key and entity as value, attach teleport-controls with said properties